pkg/console/operator/sync_v400.go

```diff
@@ -107,6 +109,15 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 	}
 	toUpdate = toUpdate || depChanged
 
+	customLogoError := co.SyncCustomLogoConfigMap(updatedOperatorConfig)
+	if customLogoError != nil {
+		msg := fmt.Sprintf("%q: %v", "customlogoconfigmap", customLogoError)
+		klog.V(4).Infof("incomplete sync: %v", msg)
+		// if the custom logo sync fails for some reason, we are degraded, not progressing.
+		co.ConditionDegraded(updatedOperatorConfig, "CustomLogoInvalid", msg)
```

Comment:
should we also call `ConditionResourceSyncNotProgressing()` ?

Reply:
I don't think so, something else could be causing a progressing error, custom logo shouldn't care about it at all.  We don't want to wipe out a progressing error set elsewhere.  But, this does illustrate why we need to improve our status reporting.  I have several ideas and a half-done PR that I'll hopefully open soon.

This just sets the one error state and the `return customLogoError` aborts the loop.  If it doesn't error, then the usual code path below calling `ConditionResourceSyncSuccess` will make everything happy.